prow/plugins.yaml

```diff
@@ -724,6 +728,10 @@ external_plugins:
     events:
       - pull_request
   kubernetes-sigs/cluster-api-provider-digitalocean:
```

Comment:
Are this and k/contrib openstack repos?

Reply:
the diff is a bit funny! (unless i messed something up). i need the plugin enabled for `kubernetes/cloud-provider-openstack` and `kubernetes-sigs/cluster-api-provider-openstack`

Reply:
oh I see it now! heh